### PR TITLE
Backend for stack restart API

### PR DESCRIPTION
**Pull request by hardillb**
## Description

Add API endpoint `/projects/<project id>/actions/restartStack`

This suspends and then restarts the project.

Useful for upgrading bare pod to Deployment (when https://github.com/flowforge/flowforge-driver-k8s/pull/71 is merged)

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass
 - [-] Documentation has been updated
    - [-] Upgrade instructions
    - [-] Configuration details
    - [-] Concepts
 - [-] Changes `flowforge.yml`?
    - [-] Issue/PR raised on `flowforge/helm` to update ConfigMap Template
    - [-] Issue/PR raised on `flowforge/CloudProject` to update values for Staging/Production

## Labels

 - [-] Backport needed? -> add the `backport` label
 - [-] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

**Comment by joepavitt** [comment]
Just checking no UI changes required for this in order to expose the action? API entry-point only?

**Comment by hardillb** [comment]
@joepavitt UI to follow later (we may hit this via a script as part of the K8s upgrade)

**Review by joepavitt — APPROVED** [review]
Form the UI side, I'm happy with the Icon-use and Verbose descriptions. I worked with Ben yesterday on assuring our logging is appropriate server-side too, but will await @knolleary approval before merging

**Review by knolleary — APPROVED** [review]

**Pull request merged**
